src/bindings/python/src/openvino/opset16/ops.py

```diff
@@ -32,3 +32,33 @@ def identity(
         as_nodes(data, name=name),
         {},
     )
+
+
+@nameable_op
+def segment_max(
+    data: NodeInput,
+    segment_ids: NodeInput,
+    num_segments: Optional[NodeInput] = None,
+    fill_mode: Union[str, None] = None,
```

Comment:
```suggestion
    fill_mode: Optional[str] = None,
```
not sure why we need union here